Python/traceback.c

```diff
@@ -705,8 +705,11 @@ extract_anchors_from_line(PyObject *filename, PyObject *line,
 
 done:
     if (res > 0) {
-        *left_anchor += start_offset;
-        *right_anchor += start_offset;
+        // Normalize the AST offsets to byte offsets and adjust it with the
+        // start of the actual line (instead of the source code segment).
```

Comment:
```suggestion
        // Normalize the AST offsets to byte offsets and adjust them with the
        // start of the actual line (instead of the source code segment).
```